vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -32,10 +33,8 @@ import (
 	"k8s.io/klog/v2"
 )
 
-// DefaultRecommenderName denotes the current recommender name as the "default" one.
-const DefaultRecommenderName = "default"
-
 var (
+	recommenderName        = flag.String("name", input.DefaultRecommenderName, "Set the recommender name for CRD filtering")
```

Comment:
What do you think about the following description for the flag:

Set the recommender name. Recommender will generate recommendations for VPAs that configure the same recommender name. If the recommender name is left as default it will also generate recommendations that don't explicitly specify recommender. You shouldn't run two recommenders with the same name in a cluster.

?